lib/appRelationsDiscovery.js

```diff
@@ -11,7 +12,7 @@ const defaultOptions = {
  * Validates and analyzes a list of AsyncAPI documents and get applications described by those files
  * 
  * @param {Array} asyncApiDocs An array of asyncApiDocuments
- * @param {Object} options for getting a relations
+ * @param {Object} options {syntax: 'syntaxName'} supportedSyntax => default, mermaid, plantUML
```

Comment:
```suggestion
 * @param {Object} [options]
 * @param {('default'|'mermaid'|'plantUML')} [options.syntax] syntax in which the relation will be generated.
```

😏

Reply:
well, am new with js docs 😅

Reply:
No worries :) I also, I always google for this shit called JSDoc :trollface: We can switch to TS in next PRs.

Reply:
never worked in TS, let's learn something new then 😉

Reply:
TS 🤦🏼